app/uk/gov/hmrc/helptosavefrontend/controllers/IvController.scala

```diff
@@ -40,7 +40,7 @@ class IvController @Inject()(val sessionCacheConnector: SessionCacheConnector,
                              frontendAuthConnector: FrontendAuthConnector)
   extends HelpToSaveAuth(app, frontendAuthConnector) with I18nSupport with Logging {
 
-  def journeyResult: Action[AnyContent] = authorisedForHts { // scalastyle:ignore cyclomatic.complexity method.length
+  def journeyResult(continueURL: String): Action[AnyContent] = authorisedForHts { // scalastyle:ignore cyclomatic.complexity method.length
```

Comment:
who will provide continueUrl  param ?
i doubt if this works , but if you have tested the IV journey locally, thats fine with me  
LGTM